packages/nimble-components/src/table-column/base/index.ts

```diff
@@ -52,6 +52,14 @@ export abstract class TableColumn<
         this.columnInternals.currentSortIndex = this.sortIndex;
     }
 
+    public checkValidity(): boolean {
+        return true;
+    }
+
+    public get validity(): ValidityObject {
```

Comment:
We're leaving it up to the author of a column type to remember to override this (and `checkValidity`) if necessary. I could see this being easy to forget.

Reply:
I'll update it in the other PR